core/src/main/scala/org/broadinstitute/dsde/rawls/billing/BillingProjectOrchestrator.scala

```diff
@@ -103,7 +92,11 @@ class BillingProjectOrchestrator(ctx: RawlsRequestContext,
 
   private def createV2BillingProjectInternal(createProjectRequest: CreateRawlsV2BillingProjectFullRequest,
                                              ctx: RawlsRequestContext
-  ): Future[Unit] =
+  ): Future[Unit] = {
+
+    val policies = BillingProjectOrchestrator.buildBillingProjectPolicies(createProjectRequest.members.getOrElse(Set.empty), ctx)
```

Comment:
Nit, defining a variable for `createProjectRequest.members.getOrElse(Set.empty)` would make this and line 111 a bit easier to read.

Reply:
Done in ae137ae13